examples/dit/args_train.py

```diff
@@ -193,6 +193,17 @@ def parse_args():
 
     parser.add_argument("--imagenet_format", type=str2bool, help="Training with ImageNet dataset format")
 
+    parser.add_argument(
```

Comment:
Do you consider setting the jit_level for sampling script `sample.py` too?

Reply:
added